src/commands/service/compile.ts

```diff
@@ -0,0 +1,73 @@
+import {readFileSync} from 'fs'
+import yaml from 'js-yaml'
+import {join} from 'path'
+
+import deployer from '../../deployer'
+import Command, {Definition} from '../../service-command'
+import MarketplacePublish from '../marketplace/publish'
+
+export default class ServiceCompile extends Command {
+  static description = 'Compile a service and upload its source to IPFS'
+
+  static flags = {
+    ...Command.flags
+  }
+
+  static strict = false
```

Comment:
not needed, this is to bypass the validation in case we support multiple arguments but we are not here
```suggestion
```